intern.json

```diff
@@ -49,6 +49,7 @@
 		"./_build/**/*.js",
 		"!./_build/**/tests/**/*",
 		"!./_build/**/styles/**/*",
+		"!./_build/**/theme/**/*",
```

Comment:
Wasn't sure if the `theme` files were supposed to be included in the coverage, I added this but can remove it if they are supposed to be in the coverage.